### PR TITLE
Changes to pass volume access point list as part of volume response

### PR DESCRIPTION
**Pull request by sameshai**

### CODE REVIEW

File: file/provider/util.go [hunk 1]
```diff
@@ -249,6 +249,51 @@ func FromProviderToLibVolume(vpcVolume *models.Share, logger *zap.Logger) (libVo
 		libVolume.Az = vpcVolume.Zone.Name
 	}
 	libVolume.CRN = vpcVolume.CRN
+
+	var respAccessPointlist = []provider.VolumeAccessPoint{}
+
+	shareTargetlist := vpcVolume.ShareTargets
+	if shareTargetlist != nil && len(*shareTargetlist) > 0 {
```
Comment (arahamad) [inline file/provider/util.go:256]: Can we have separate if condition for these checks and for loop outside of look

File: file/provider/util.go [hunk 2]
```diff
@@ -249,6 +249,51 @@ func FromProviderToLibVolume(vpcVolume *models.Share, logger *zap.Logger) (libVo
...
+		}
+	}
+
+	libVolume.VolumeAccessPoints = &respAccessPointlist
+	return
+}
+
+// FromProviderToLibVolumeAccessPoint converting vpc provider share target type to generic lib volume accessPoint Type
+func FromProviderToLibVolumeAccessPoint(vpcShareTarget *models.ShareTarget, logger *zap.Logger) (libVolumeAccessPoint *provider.VolumeAccessPoint) {
+	logger.Debug("Entry of FromProviderToLibVolumeAccessPoint method...")
```
Comment (arahamad) [inline file/provider/util.go:269]: debug will never be shown in the log can be add as info

**Review by arahamad — COMMENTED** [review]
please check if we can improve

File: file/provider/util.go (hunk 2 above)
Comment (sameshai) [reply file/provider/util.go:269]: This pattern is followed in block so carried over. I though we don't need to show this entry exit until we want to enable debug.

File: file/provider/util.go (hunk 1 above)
Comment (sameshai) [reply file/provider/util.go:256]: There is no else part for this logic so we have to populate the list if only we have share targets. Let me know if you some other flow in mind

**Review by arahamad — APPROVED** [review]
LGTM

File: common/messages/messages_en.go [hunk 3]
```diff
@@ -74,7 +74,7 @@ var messagesEn = map[string]util.Message{
 	},
 	"AccessPointWithAPIDFindFailed": {
 		Code:        AccessPointWithAPIDFindFailed,
-		Description: "No volume access point could not be found for the specified volume ID '%s' and access point ID '%s' or VPC ID %s",
+		Description: "No volume access point could not be found for the specified volume ID '%s' and access point ID '%s'",
```
Comment (meghnasiingh) [inline common/messages/messages_en.go:77]: This line doesn't looks to be correct. Instead we can have Volume access point could not be found for the specified volume ID '%s' with access point ID '%s'

Comment (sameshai) [reply common/messages/messages_en.go:77]: I can do that but this standard message was followed to keep consistency between block library.
"	"VolumeAttachFindFailed": {
		Code:        VolumeAttachFindFailed,

		Description: "No volume attachment could be found for the specified volume ID '%s' and instance ID '%s'.",
		Type:        util.VolumeAttachFindFailed,
		RC:          400,
		Action:      "Verify that a volume attachment for your instance exists. Run 'ibmcloud is in-vols INSTANCE_ID' to list active volume attachments for your instance ID. ",
	},"

**Review by meghnasiingh — APPROVED** [review]

**Pull request merged**